server.js

```diff
@@ -26,13 +28,29 @@ function handleGetLocation(req, res) {
   res.send(output);
 }
 
+function handleGetWeather(req, res) {
+
+  const data = require('./data/weather.json');
+  console.log(data.data[0].datetime);
+
+  const output = new Weather (data.data[0].weather.description, data.data[0].datetime);
+  console.log(output);
+
+  res.send(output);
+}
+
 function Location(dataFromTheFile, cityName) {
   this.search_query = cityName;
   this.fomatted_query = dataFromTheFile[0].display_name;
   this.latitude = dataFromTheFile[0].lat;
   this.longitude = dataFromTheFile[0].lon;
 }
 
+function Weather(description, time) {
+  this.forecast = description;
+  this.time = time;
+}
+
 //_______Initialization______
 
 app.listen(PORT, () => { console.log('app is up on http://localhost:3009');});
```

Comment:
I would change it to `app is up on port http://localhost:${PORT}`. So your console.log() will be more flexible